app/routes.js

```diff
@@ -276,31 +197,43 @@ export const getRoutes = (appContext, store) => {
   let props = appContext.props;
   let api = props.api;
 
+  let boundRequireNoAuth = requireNoAuth.bind(null, api);
```

Comment:
Small nitpick.  These, and the existing variables above are never modified, and I'm thinking should be changed to `const` instead of `let`

Reply:
Yup, `let` was a vestige of an earlier iteratio.n